crml/staking/src/tests.rs

```diff
@@ -3226,20 +3181,22 @@ fn payout_to_any_account_works() {
 		bond_nominator(1234, 1337, 100, vec![11]);
 
 		// Update payout location
-		assert_ok!(Staking::set_payee(Origin::signed(1337), RewardDestination::Account(42)));
+		assert_ok!(Staking::set_payee(Origin::signed(1337), RewardDestination::Account(41)));
```

Comment:
why change the account values here, seems ok as it was?

Reply:
Account 42 is not created in the genesis config. When paying rewards we do deposit_into_existing which means if the account does exist we pay the reward. It was okay before because in the function make_payout for the mocker we had instead done deposit_creating. But now instead of the mock we are using the actual Rewards code.

Reply:
ok this seems like an issue in the reward function since we don't limit the reward destination to existing accounts only

Reply:
Do you mean in Staking::set_payee and Staking::bond, we should check if the destination account exists? This is a good idea. However, we still need to change 42 to 41 in this test.

Reply:
The reward function issue is fixed according to our chat on slack.

Reply:
I think this can change back now after the `deposit_creating` change

Reply:
Done